schema/src/main/scala/io/shiftleft/codepropertygraph/schema/Method.scala

```diff
@@ -38,6 +38,7 @@ object Method extends SchemaBase {
                     |implementor to fit the source language.
                     |""".stripMargin
       )
+      .mandatory(PropertyDefaults.String)
```

Comment:
We could consider empty-string for this one

Reply:
I agree, changed.